src/state/reducers/index.js

```diff
@@ -0,0 +1,11 @@
+// place for reducers connection
+import { combineReducers } from 'redux';
+import { recipesReducer } from './recipesReducer';
+import { favoritesReducer } from './favoritesReducer';
+import { fetchingRecipesReducer } from './fetchingRecipesReducer';
+
+export const rootReducer = combineReducers ({
+  recipes: recipesReducer,
+  favorites: favoritesReducer,
+  fetchingRecipes: fetchingRecipesReducer
```

Comment:
This reducer seems to be redundant. If you want to keep an information about `loading` you can keep it in `recipesReducer`